src/Sentry/Internal/Hub.cs

```diff
@@ -394,7 +396,10 @@ public SentryId CaptureEvent(SentryEvent evt, SentryHint? hint, Action<Scope> co
             var clonedScope = CurrentScope.Clone();
             configureScope(clonedScope);
 
-            return CaptureEvent(evt, clonedScope, hint);
+            // Although we clone a temporary scope for the configureScope action, for the second scope
+            // argument (the breadcrumbScope) we pass in the current scope... this is because we want
+            // a breadcrumb to be left on the current scope for exception events
```

Comment:
I guess adding the breadcrumb here is not an option. They are both public and the other `CaptureEvent` is the last down the chain, right? 🥲